setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+import digits
```

Comment:
this is the script to install DIGITS through `pip`. We need to do this as plug-ins reference some module within DIGITS and they need to do so using absolute imports